.github/workflows/test.yml

```diff
@@ -201,6 +201,14 @@ jobs:
           args: --all --all-features -- -D warnings
           token: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Install Foundry
+        uses: onbjerg/foundry-toolchain@v1
+        with:
+          version: nightly
+      
+      - name: forge fmt
+        run: forge fmt --check testdata/
+
```

Comment:
cool!

do we have this in the book already?

Reply:
not yet, there's only an issue https://github.com/foundry-rs/book/issues/38